src/main/pg/bus_spi.c

```diff
@@ -71,6 +71,7 @@ PG_REGISTER_ARRAY(spiCs_t, SPI_PREINIT_OPU_COUNT, spiPreinitOPUConfig, PG_SPI_PR
 // Generic target will be specifying both values by resource commands.
 
 ioTag_t preinitIPUList[SPI_PREINIT_IPU_COUNT] = {
+#ifndef GENERIC_TARGET
```

Comment:
Soon all (except for some 'exotic') targets will be generic or at least unified. If this is not to be used for these, then this should be completely removed - there is no point in dragging non-functional legacy code along, it will only end up wasting resources for maintenance without providing value

Reply:
I think the code is required as soon multiple devises sharing one SPI bus. @jflyper you have an idea how this can be properly fixed?

Reply:
If it is required in any case, then simply not building it for generic targets is not a solution.

Re doing this in a separate pull request. This pull request seems to try to make things ready for generic targets. If it does not do that, then it should probably defer doing the changes until we've figured out a way to do this in a way that works.

Reply:
Some SPI devices have internal pull-ups at the CS pins and in this case it will work with floating pins from STM32 side and the devices will be detected properly. This looks to be the case for the various AFNGF7 configurations. But there where some issues in the past. Lets see if @jflyper has an good idea i think he designed this initially if i remember well?